src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,180 @@
+# This is a rough draft feature for future use.
+
+Feature: Resources "Delete" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
```

Comment:
A `When` statement should not show up in the Background. Please change to `And`.

Reply:
Changed